facebook_connector.py

```diff
@@ -0,0 +1,40 @@
+from fbchat_archive_parser.parser import MessageHtmlParser
+import fnmatch
+import os
+import pandas as pd
+
+
+def initialize(dump_location='.'):
+    global messages_file
```

Comment:
Add `messages_file = None` at the top of this file to introduce the variable and to indicate that it'll exist later.  Also name it `_messages_file` to indicate that it's private and not used outside this module